src/plugins/filedelete/filedelete.cpp

```diff
@@ -514,6 +529,7 @@ static void register_trap( drakvuf_t drakvuf, const char* rekall_profile, const
 }
 
 filedelete::filedelete(drakvuf_t drakvuf, const void* config, output_format_t output)
+    : sequence_number()
```

Comment:
My C++ foo is not strong enough, is this a 0-initialization? :)

Reply:
Yes.